docs/source/en/optimization/attention_backends.md

```diff
@@ -78,10 +78,16 @@ with attention_backend("_flash_3_hub"):
     image = pipeline(prompt).images[0]
 ```
 
+> [!TIP]
+> Most of these attention backends come with `torch.compile` compatibility without any graph breaks. Consider using it for maximum speedups.
+
 ## Available backends
 
 Refer to the table below for a complete list of available attention backends and their variants.
 
+<details>
```

Comment:
May be simpler to replace the table at the beginning of the docs with this more complete one, wdyt?

Reply:
Actually the first table is nice in the sense that it provides a consolidated overview of what's supported. So, I'd prefer to keep it.